_main.scss

```diff
@@ -0,0 +1,55 @@
+@use './helpers/mixnes' as *;
+@use './variables/variables' as *;
```

Comment:
variables must be included first thing, so mixins will have access to it

Reply:
main.scss is the one that contains the styles, not the assets, move all the assts (variables, mixins) to index.scss

and after including all assets, include main.scss which uses these assets and defines new styles